container_pipeline/workers/scanners/scan.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python
+
+import docker
+import json
+import logging
+import os
+import shutil
+import subprocess
+import config
+import hashlib
+
+from Atomic import Atomic, mount
+
+from container_pipeline.utils import Build, get_job_name
+from container_pipeline.lib.log import load_logger
+from container_pipeline.workers.base import BaseWorker
+from container_pipeline.workers.scanners.runner import ScannerRunner
+
+class ScanWorker(BaseWorker):
+    """Scan worker"""
+
+    def handle_job(self, job):
+        """
+        This scans the images for the job requests in start_scan tube.
+        this cals the ScannerRunner for performing the scan work
+        """
+
+        debug_logs_file = os.path.join(
+            job_info["logs_dir"], "service_debug.log")
+        dfh = config.DynamicFileHandler(logger, debug_logs_file)
```

Comment:
This is not needed - neither one should hard code the `service_debug.log` name (for future reference, it is there in `default_settings.py`)